lib/sof/runner.rb

```diff
@@ -84,7 +84,7 @@ def output_results(jira_format, verbose = false)
 
     failure_results.sort!{ |a,b| a.first <=> b.first }
     failure_results.each do |result|
-      puts result[0].colorize(:red)
+      puts result[0].colorize(:red) if STDOUT.isatty
```

Comment:
These results won't be printed at all if STDOUT is not a TTY.  I'm not sure thats what you're trying to do.